app/uk/gov/hmrc/hec/services/FileCreationService.scala

```diff
@@ -183,7 +183,7 @@ class FileCreationServiceImpl @Inject() (timeProvider: TimeProvider) extends Fil
             notChargeable = taxSituationMap.NotChargeable,
             PAYE = taxSituationMap.PAYE,
             SA = taxSituationMap.SA,
-            incomeTaxYear = i.taxDetails.relevantIncomeTaxYear.some,
+            incomeTaxYear = i.taxDetails.relevantIncomeTaxYear.startYear.some,
```

Comment:
this should have been the start year plus 1 before according to the spec - they want the end year. Could we fix it here?

Reply:
yeah sure

Reply:
[updated](https://github.com/hmrc/hec/pull/45/commits/54e1f337e21674b5a8f8031e5a7529ad715982f2)